components/Login/index.tsx

```diff
@@ -1,13 +1,41 @@
-import { FC } from "react";
-import { WebsiteUrls } from "../../types/enums";
+import axios from "axios";
+import { observer } from "mobx-react-lite";
+import { NextRouter, useRouter } from "next/router";
+import { FC, useState } from "react";
+import userStore from "../../store/userStore";
+import { BaseUrl, WebsiteUrls } from "../../types/enums";
+import { Values } from "../../types/formik";
 import LoginComponent from "../LoginComponent";
 
-const Login: FC = () => (
-  <LoginComponent
-    heading="Sign in"
-    link="Sign in with Social"
-    href={WebsiteUrls.SOCIAL}
-  />
-);
+const Login: FC = () => {
+  const [exist, setExist] = useState(false);
+  const router: NextRouter = useRouter();
 
-export default Login;
+  const submitHandler = async (values: Values) => {
+    try {
+      const { data } = await axios.post(`${BaseUrl.URL}auth/login`, values);
+
+      userStore.addUser(data.user);
+
+      data.message === "user not found with this email"
```

Comment:
Can we check status code (404) instead of plain string? This string could be changed and your code won't work.